### ISSUE TITLE
Missing arguments in vgg_loss function

### ISSUE THREAD

**Issue by YolCruz** [deepak112/Keras-SRGAN#17]
When you make the class VGG_LOSS, the function vgg_loss takes two arguments: y_true and y_pred, but when you call this astribute in the line 52 and 55 of train.py, you do not give those arguments. I'm getting an error from this missing arguments, what can I do to solve it?

**Comment by mrciolino** [comment]
i think he is passing an function as an object to keras to use that function as the loss function. Could you show the code you are using and the error?

**Comment by BassantTolba1234** [comment]
Please all,
I need the code of implementation this part 
the part is 

{The SRResNet networks
were trained with a learning rate of 10−4 and 106 update
iterations. We employed the trained MSE-based SRResNet
network as initialization for the generator when training
the actual GAN to avoid undesired local optima.{
